packages/grafana-prometheus/src/querybuilder/components/MetricCombobox.tsx

```diff
@@ -171,3 +176,24 @@ const formatKeyValueStringsForLabelValuesQuery = (query: string, labelsFilters?:
 
   return formatPrometheusLabelFiltersToString(queryString, labelsFilters);
 };
+
+const getMectricComboboxStyles = () => ({
+  addaptToParentQueryEditor: css({
+    // Take metrics explorer button into account
+    maxWidth: `calc(100% - 32px)`,
+  }),
+  addaptToParentVariableEditor: css({
+    maxWidth: '100%',
+    display: 'flex',
+    '[class*="InlineFieldRow"]': {
+      maxWidth: '100%',
+      '> div': {
+        maxWidth: '100%',
```

Comment:
Should these changes be set in the `InlineFieldRow` component instead of having nested electors here?